docs/manage/users-roles/roles/role-capabilities.md

```diff
@@ -5,379 +5,177 @@ description: Assign any of these capabilities when you create user roles.
-   </td>
-  </tr>
-  <tr>
-   <td>Create Trial Organizations
-   </td>
-   <td>Create trial organizations. (For Sumo Logic Service Providers only.)
-   </td>
-  </tr>
-  <tr>
-   <td>Upgrade Trial Organizations
-   </td>
-   <td>Upgrade trial organizations. (For Sumo Logic Service Providers only.)
-   </td>
-  </tr>
-  <tr>
-   <td>Deactivate Organizations
-   </td>
-   <td>Deactivate trial organizations. (For Sumo Logic Service Providers only.)
-   </td>
-  </tr>
-</table>
+| Capability | Description |
+| :-- | :-- |
+| View Organizations	| View the [Organizations](/docs/manage/manage-subscription/create-manage-orgs) UI.| 
+| Create Organizations	| Create and provision child organizations.| 
+| Change Credits Allocation	| Change the credits allocation for a child organization.| 
+| Create Trial Organizations	| Create trial organizations. (For Sumo Logic Service Providers only.)| 
+| Upgrade Trial Organizations	| Upgrade trial organizations. (For Sumo Logic Service Providers only.)| 
+| Deactivate Organizations	| Deactivate trial organizations. (For Sumo Logic Service Providers only.)| 
+
+## Cloud SOAR
+
+[Cloud SOAR](/docs/cloud-soar/) capabilities only appear in the Roles UI if Cloud SOAR has been enabled for your account.
+
+| Capability category | Capability | Description |
+| :-- | :-- | :-- |
+| View Cloud SOAR | | Users with a role that grants this capability will see a **Cloud SOAR** link in the left-nav bar of the Sumo Logic UI. | 
+| App Central | Access | Access [App Central](/docs/cloud-soar/automation#app-central). |  
+| App Central | Export | Export from App Central. |  
+| Integrations | Access | Access [Integrations](/docs/cloud-soar/automation#integrations). |  
+| Integrations | Configure | Configure Integrations. |  
+| Automation Playbooks | Access | Access [playbooks](/docs/cloud-soar/automation#playbook).  |  
+| Automation Playbooks | Configure | Configure playbooks. |  
+| Bridge Monitoring | Access | Access [Bridge monitoring](/docs/cloud-soar/global-functions-menu/#audit-and-information). | 
+| Observability | Access | Access [Incidents](/docs/cloud-soar/incidents-triage/).  |   
+| Observability | Configure | Configure Incidents. |   
+
+
+## Cloud SIEM Enterprise
+
+[Cloud SIEM Enterprise](/docs/cse/) capabilities only appear in the Roles UI if Cloud SIEM has been [enabled](/docs/cse/get-started-with-cloud-siem/onboarding-checklist-cse/#provision-cloud-siem) for your account. For more information about how to assign Cloud SIEM capabilities, see [Cloud SIEM User Accounts and Roles](/docs/cse/administration/cse-user-accounts-and-roles/).
+
+| Capability category | Capability | Description |
+| :-- | :-- | :-- |
+| View Cloud SIEM Enterprise | | Users with a role that grants this capability will see a **Cloud SIEM Enterprise** link in the left-nav bar of the Sumo Logic UI. When a user clicks on the link, the Cloud SIEM [Heads-Up Display (HUD)](/docs/cse/get-started-with-cloud-siem/cse-heads-up-display/) will open. |
```

Comment:
The middle column is blank - just double-checking if that's intentional

Reply:
Yes, @kimsauce, that's intentional. That makes it match the UI.